Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 GIT
   remote: https://github.com/wordpress-mobile/release-toolkit
-  revision: eb0f24ec6c73b3454d711cc4b3276c50b9655861
-  tag: 0.18.1
+  revision: 8534f11250be45e2b848b9ccbeea6b474a9b63d3
+  branch: allow-paths-outside-project-folder
```

Comment:
Waiting for https://github.com/wordpress-mobile/release-toolkit/pull/252 to be merged.